crates/rust-client/src/transaction/mod.rs

```
@@ -1159,6 +1140,18 @@ impl Client {
             )
             .await?)
     }
+
+    pub(crate) fn build_executor<'store, 'auth>(
+        &'auth self,
+        data_store: &'store ClientDataStore,
+    ) -> TransactionExecutor<'store, 'auth> {
+        TransactionExecutor::with_options(
+            data_store,
+            self.authenticator.as_deref(),
+            self.exec_options,
+        )
+        .unwrap()
```

We should surface this error

---

Totally, forgot about this unwrap while testing. Nice catch, thanks